styles.css

```diff
@@ -92,21 +109,90 @@ Text {
 .Text {
   color: white;
 }
-.Icon {
-  background-image: url("https://i.ibb.co/C6nmMvx/AB5-EAFE4-AC20-4-E6-D-BFD0-EF953-C9818-A0.png")
-    no-repeat;
-  border-radius: 50%;
-  width: 40px;
-  height: 40px;
-  margin: 10px;
+.Button {
+  background-color: #171f29;
+  display: flex;
+  align-items: center;
+  border-top-left-radius: 50px;
+  border-bottom-left-radius: 50px;
+  border-top-right-radius: 50px;
+  border-bottom-right-radius: 50px;
```

Comment:
This is equivalent to `border-radius: 50px`. Generally, you would only call out each corner separately if they differ from each other.